projects/storefrontstyles/scss/components/content/navigation/_scroll-to-top.scss

```diff
@@ -0,0 +1,49 @@
+%cx-scroll-to-top {
+  display: none;
+  height: 50px;
+  width: 50px;
+  position: fixed;
+  bottom: 60px;
+  right: 30px;
+
+  &.display {
+    display: initial;
```

Comment:
Why use `initial` in this case and not the value we expect for the button to display?  Let's discuss, it's mostly for my understanding.

Reply:
Yes let's discuss